glom/test/test_error.py

```diff
@@ -60,20 +60,6 @@ def test_line_trace():
     assert fmtd_stack == '/tuple!list/list<0>/dict!int/S'
 
 
-def test_short_trace():
```

Comment:
curious why this one got removed.

Reply:
I think only_errors got added to _unpack_stack() trim "extra" branches, but in the process it kind of broke this test case -- it relies on extracting the scope from inside an executing glom that then completes and formatting it

this isn't really intended to be possible

also, this was giving low value vs the ones that actually use a raised exception